base/shell/explorer/traywnd.cpp

```diff
@@ -159,13 +163,6 @@ IsThereAnyEffectiveWindow(BOOL bMustBeInMonitor)
     ei.bMustBeInMonitor = bMustBeInMonitor;
 
     EnumWindows(FindEffectiveProc, (LPARAM)&ei);
-    if (ei.hwndFound && FALSE)
-    {
-        WCHAR szClass[64], szText[64];
-        GetClassNameW(ei.hwndFound, szClass, _countof(szClass));
-        GetWindowTextW(ei.hwndFound, szText, _countof(szText));
-        MessageBoxW(NULL, szText, szClass, 0);
-    }
     return ei.hwndFound != NULL;
```

Comment:
Do I assume correctly, that the deletion of that block above has nothing to do with the fix, but is unrelated cleanup?

Reply:
`((...) && FALSE)` means `FALSE`.
The above code will be ignored and useless.